android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,11 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.novoda.android">
+
+    <application android:allowBackup="true"
+        android:label="@string/app_name"
+        android:supportsRtl="true"
```

Comment:
as in, if a downstream person now _has_ to set it false (and it isn't false by default) that should be mentioned in the release notes when we come to release it :)